types/upnp-mediarenderer-client/upnp-mediarenderer-client-tests.ts

```diff
@@ -0,0 +1,81 @@
+import MediaRendererClient from 'upnp-mediarenderer-client';
```

Comment:
Consider using `import MediaRendererClient = require('upnp-mediarenderer-client');` here, rather than `esModuleInterop` in the tsconfig.

Reply:
changed